### PR TITLE
Add AppVeyor Config

### PR DESCRIPTION
**Pull request by AustinShalit**

### CODE REVIEW

**Comment by JLLeitschuh** [comment]
I would use the wpilib appveyor account if you can instead of your own.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/wpilibsuite/shuffleboard.svg?branch=master)](https://travis-ci.org/wpilibsuite/shuffleboard)
+[![Build Status](https://travis-ci.org/wpilibsuite/shuffleboard.svg?branch=master)](https://travis-ci.org/wpilibsuite/shuffleboard) [![Build status](https://ci.appveyor.com/api/projects/status/auljw926o10sea4w/branch/master?svg=true)](https://ci.appveyor.com/project/AustinShalit/shuffleboard/branch/master)
```
Comment (JLLeitschuh) [inline README.md]: You can put these on separate lines. Without an extra line break they will render on the same line.

**Comment by AustinShalit** [comment]
I will talk with @bradamiller about getting it setup under another account.

File: README.md (hunk 1 above)
Comment (AustinShalit) [inline README.md]: Sounds good

Comment (JLLeitschuh) [inline README.md]: You should do this before merging

**Comment by AustinShalit** [comment]
#108

**Pull request merged**